src/components/checkbox/index.tsx

```diff
@@ -1,56 +0,0 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { Checkbox } from "antd";
-import "../../App.css";
-interface CheckboxProps {
-  checkboxNumber: number;
-}
-
-const CustomCheckbox = ({ checkboxNumber }: CheckboxProps) => {
-  const [letters, setLetters] = useState<string[]>([]);
-  const [checkvalue, setCheckValue] = useState<boolean>(false);
-  const url =
-    "https://navirego-interview-mc3narrsb-volodymyr-matselyukh.vercel.app/api/letters/";
-  useEffect(() => {
-    const handleFetch = async () => {
-      const res = await axios.get(`${url}${checkboxNumber}`);
```

Comment:
The app doesn't handle any sort of error which might come up, suppose the API throws 401 error or any kind of BAD request or too many request, these things has to be captured by the axios interceptor which we declare in the http.ts file in the utils. 

Read about interceptors and how they work. This is useful for other projects too.